game.gd

```diff
@@ -1,35 +1,31 @@
 extends Node2D
+class_name Game
 
 @onready var other: CharacterBody2D = $Other
 @onready var ball: RigidBody2D = $Ball
 @onready var other_shape: CollisionShape2D = $Other/CollisionShape2D
 
-@onready var other_label : Label = $OtherScore
-@onready var player_label : Label = $PlayerScore
-
 @onready var hit_audio_player: AudioStreamPlayer = $HitAudioPlayer
 @onready var rebound_audio_player: AudioStreamPlayer = $ReboundAudioPlayer
-@onready var menu: Menu = $Menu
 
 
 var ball_speed := 300
 const OTHER_SPEED = 300
 const OTHER_MIDDLE = 128/2
 const MOVE_LIMIT = 25 # amount of pixels to not move (prevent AI stutters)
-var paused := false
 
-var other_score := 0
-var player_score := 0
+var other_score: int = 0
+var player_score: int = 0
```

Comment:
Although it's obvious 0 is an integer I took this habit to avoid problems while trying to do conversion between ints and floats.

But in this case it should be fine to not use the types.